src/UglyToad.PdfPig/Graphics/IOperationContext.cs

```diff
@@ -3,13 +3,15 @@
     using PdfPig.Core;
     using System.Collections.Generic;
     using Tokens;
-    using Util.JetBrains.Annotations;
+    using UglyToad.PdfPig.Graphics.Core;
+    //using Util.JetBrains.Annotations;
```

Comment:
Yeah we can remove this reference, it was basically obsoleted by C# 8 whenever we can start using it :D